src/NServiceBus.Core.Analyzer.Tests/UniformSessionStubs/IUniformSessionExtensions.cs

```diff
@@ -1,8 +1,10 @@
 ﻿namespace NServiceBus.UniformSession
 {
     using System;
+    using System.Diagnostics.CodeAnalysis;
     using System.Threading.Tasks;
 
+    [SuppressMessage("Code", "PCR0019:A task-returning method should have a CancellationToken parameter or a parameter implementing ICancellableContext", Justification = "<Pending>")]
```

Comment:
The UniformSession package itself has been updated for tokens. Should these stubs be updated to match?